js/src/admin.js

```diff
@@ -2,425 +2,6 @@
-
-										// Make notices dismissible
-										// copy paste of common.js from WP 4.2.2
-										$( '.notice.is-dismissible' ).each(
-											function() {
-												var $this = $( this ),
-													$button = $( '<button type="button" class="notice-dismiss"><span class="screen-reader-text"></span></button>' ),
-													btnText = pll_admin.dismiss_notice || '';
-
-												// Ensure plain text
-												$button.find( '.screen-reader-text' ).text( btnText );
-
-												// Whitelist because of how the button is built. See above
-												$this.append( $button ); // phpcs:ignore WordPressVIPMinimum.JS.HTMLExecutingFunctions.append
-
-												$button.on(
-													'click.wp-dismiss-notice',
-													function( event ) {
-														event.preventDefault();
-														$this.fadeTo(
-															100,
-															0,
-															function() {
-																$( this ).slideUp(
-																	100,
-																	function() {
-																		$( this ).remove();
-																	}
-																);
-															}
-														);
-													}
-												);
-											}
-										);
-									break;
-								}
-							}
-						);
-					}
-				);
-			}
-		);
-
-		// act when pressing enter or esc in configurations
-		$( '.pll-configure' ).on(
-			'keydown',
-			function( event ){
-				if ( 'Enter' === event.key ) {
-					event.preventDefault();
-					$( this ).find( '.save' ).trigger( 'click' );
-				}
-
-				if ( 'Escape' === event.key ) {
-					event.preventDefault();
-					$( this ).find( '.cancel' ).trigger( 'click' );
-				}
-			}
-		);
-
-		// settings URL modifications
-		// manages visibility of fields
-		$( "input[name='force_lang']" ).on(
-			'change',
-			function() {
-				function pll_toggle( a, test ) {
-					test ? a.show() : a.hide();
-				}
-
-				var value = $( this ).val();
-				pll_toggle( $( '#pll-domains-table' ), 3 == value );
-				pll_toggle( $( "#pll-hide-default" ), 3 > value );
-				pll_toggle( $( "#pll-rewrite" ), 2 > value );
-				pll_toggle( $( "#pll-redirect-lang" ), 2 > value );
-			}
-		);
-
-		// settings license
-		// deactivate button
-		$( '.pll-deactivate-license' ).on(
-			'click',
-			function() {
-				var data = {
-					action:            'pll_deactivate_license',
-					pll_ajax_settings: true,
-					id:                $( this ).attr( 'id' ),
-					_pll_nonce:        $( '#_pll_nonce' ).val()
-				};
-				$.post(
-					ajaxurl,
-					data,
-					function( response ){
-						$( '#pll-license-' + response.id ).replaceWith( response.html );
-					}
-				);
-			}
-		);
-
-		// Manage closing the metabox.
-		// close postboxes that should be closed
-		$( '.if-js-closed' ).removeClass( 'if-js-closed' ).addClass( 'closed' );
-		// postboxes setup
-		if ( 'undefined' !== typeof postboxes ) {
-			postboxes.add_postbox_toggles( pagenow );
-		}
-	}
-);
+import ajaxFilter from './lib/ajax-filter.js';
 
+ajaxFilter( pll_admin.ajax_filter );
```

Comment:
```suggestion
ajaxFilter( pll_admin?.ajax_filter );
```
If someone mess around our code in PHP.

Reply:
Done in [587484f](https://github.com/polylang/polylang/pull/1612/commits/587484f4d8d9a8638d92cb00b86246ce36075b6a).